dojo/models.py

```diff
@@ -481,7 +481,7 @@ def calc_health(self):
     # only used by bulk risk acceptance api
     @property
     def unaccepted_open_findings(self):
-        return Finding.objects.filter(risk_accepted=False, active=True, verified=True, duplicate=False, test__engagement__product__prod_type=self)
+        return Finding.objects.filter(risk_accepted=False, active=True, verified=False, duplicate=False, test__engagement__product__prod_type=self)
```

Comment:
now you require the findings to be NOT verified. so it will skip verified findings :-)

Reply:
good eye :facepalm: